### PR TITLE
[COM-819] Integrate SignalWire API to send SMS messages

### PR DESCRIPTION
**Pull request by Angele91**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)

https://impulsumvc.atlassian.net/browse/COM-819

### CODE REVIEW

File: packages/signalwire/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# @cmpsr/twilio
```
Comment (tkanzakic) [inline packages/signalwire/README.md:1]: please update all twilio references

File: packages/signalwire/config/jest/cssTransform.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (tkanzakic) [inline packages/signalwire/config/jest/cssTransform.js:1]: is this needed?

File: packages/signalwire/config/jest/setupTests.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+jest.mock('@signalwire/core', () => {
```
Comment (tkanzakic) [inline packages/signalwire/config/jest/setupTests.ts:8]: can we get rid of this mock?, not sure if it is still needed

File: packages/signalwire/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "/lib",
+    "package.json"
+  ],
+  "scripts": {
+    "libbuild": "rimraf lib && npx swc ./src -d lib -s --ignore '**/*.test.ts'",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:coverage": "jest --coverage",
+    "type:check": "yarn tsc --project tsconfig.json --noEmit",
+    "type:emit": "tsc --project ./tsconfig.json -d --declarationDir lib --declarationMap --emitDeclarationOnly && find lib -type f \\( -name '*.test.*' -o -name '*.stories.*' \\) -exec rm {} +"
```
Comment (tkanzakic) [inline packages/signalwire/package.json:25]: I think we can remove the `'*.stories.*'`  cleanup from here

**Review by tkanzakic — COMMENTED** [review]
This looks great, just a couple of minor comments

File: packages/signalwire/README.md (hunk 1 above)
Comment (Angele91) [reply packages/signalwire/README.md]: Done!

File: packages/signalwire/config/jest/cssTransform.js (hunk 2 above)
Comment (Angele91) [reply packages/signalwire/config/jest/cssTransform.js]: Done!

File: packages/signalwire/config/jest/setupTests.ts (hunk 3 above)
Comment (Angele91) [reply packages/signalwire/config/jest/setupTests.ts]: Removed!

**Review by tkanzakic — APPROVED** [review]
LGTM 🚀 !

**Pull request merged**